idc/settings.py

```diff
@@ -295,6 +295,7 @@ def GET_BQ_COHORT_SETTINGS():
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'request_logging.middleware.LoggingMiddleware',
     'offline.middleware.OfflineMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
 ]
```

Comment:
This is an unnecessary edition, as the SessionMiddleware is already enabled in this list (3rd from the top).